src/bridge.ts

```diff
@@ -43,7 +43,7 @@ export const initializeRoutes = (app: express.Server, serviceUrl: string, botUrl
 
     //Gets activities from store (local history array for now)
     app.get('/directline/conversations/:conversationId/activities', (req, res) => {
-        let watermark = Number(req.query.watermark || 0);
+        let watermark = req.query.watermark && req.query.watermark !== "null" ? Number(req.query.watermark) : 0;
```

Comment:
@User1m I can get behind this change! The other changes in this file seem like different spacing (to comments, etc).

Reply:
@ryanvolum yes. Will do

Reply:
gotcha - you using vscode? I thought I had everything up to its (standard) standards...